sqlite/src/analyze.c

```diff
@@ -1683,7 +1698,9 @@ static void analyzeDatabase(Parse *pParse, int iDb){
     Table *pTab = (Table*)sqliteHashData(k);
     analyzeOneTable(pParse, pTab, 0, iStatCur, iMem, iTab);
   }
+#if !defined(SQLITE_BUILDING_FOR_COMDB2)
```

Comment:
Calling the analyze loader here does not seem quite right.  I suspected the #ifdef was missing.  Double check please.